### PR TITLE
basic-features.md: add the column for v7.6.0

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

Add the column for v7.6.0

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,235 +20,235 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [TiFlash 副本选择策略](/system-variables.md#tiflash_replica_read-从-v730-版本开始引入) |  Y  | N | N | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md) |  Y  | Y | Y | Y | Y | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) |  Y  | Y | Y | Y | E | E | N | N | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) |  E  | E | E | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) |  Y  | N | N | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) |  Y  |  Y  | Y | Y | Y | Y | Y | E | E | E | E |
+| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) |  Y  |  Y  | E | N | N | N | N | N | N | N | N |
+| [执行计划管理 (SPM)](/sql-plan-management.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
[the remark this answers is not in the document]
Comment (songrijie) [reply basic-features.md:96]: LGTM

File: basic-features.md [hunk 2]
```diff
@@ -20,235 +20,238 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [读已提交隔离](/transaction-isolation-levels.md) |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 事务 | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Async commit](/system-variables.md#tidb_enable_async_commit-从-v50-版本开始引入) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [1PC](/system-variables.md#tidb_enable_1pc-从-v50-版本开始引入) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [大事务 (10 GB)](/transaction-overview.md#事务限制) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [悲观事务](/pessimistic-transaction.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [乐观事务](/optimistic-transaction.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [可重复读隔离（快照隔离）](/transaction-isolation-levels.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [读已提交隔离](/transaction-isolation-levels.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (benmaoer) [inline basic-features.md:144]: | [`tidb_idle_transaction_timeout`](/system-variables.md#tidb_idle_transaction_timeout-从-v760-版本开始引入) | Y | N | N | N | N | N | N | N | N | N | N |

**Review by benmaoer — CHANGES_REQUESTED** [review]
add idle transaction timeout

**Review by benmaoer — CHANGES_REQUESTED** [review]
add idle txn timeout

File: basic-features.md (hunk 2 above)
Comment (Oreoxmt) [inline basic-features.md:144]:
```suggestion
| [读已提交隔离](/transaction-isolation-levels.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
| [自动终止长时间未提交的空闲事务](/system-variables.md#tidb_idle_transaction_timeout-从-v760-版本开始引入) | Y | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 3]
```diff
@@ -20,235 +20,239 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [快速导入 (`IMPORT INTO`)](/sql-statements/sql-statement-import-into.md) | Y | Y | N | N | N | N | N | N | N | N | N |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) [^5] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^6] |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) [^7] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiCDC 支持保存数据到存储服务 (Amazon S3/GCS/Azure Blob Storage/NFS)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | Y | Y | E | N | N | N | N | N | N | N |
+| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | Y | Y | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:218]:
```suggestion
| [DM](/dm//dm-overview.md) 支持迁移 MySQL 8.0  | Y | E | E | E | E | N | N | N | N | N | N |
```

File: basic-features.md [hunk 4]
```diff
@@ -20,235 +20,239 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [TTL (Time to Live)](/time-to-live.md) |  Y  |  Y  | Y | E | N | N | N | N | N | N | N |
+| [DDL 算法断言](/sql-statements/sql-statement-alter-table.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 在单条语句中添加多列 |  Y  |  Y  | Y | Y | E | E | E | E | E | E | E |
+| [更改列类型](/sql-statements/sql-statement-modify-column.md) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [临时表](/temporary-tables.md) |  Y  |  Y  | Y | Y | Y | Y | Y | N | N | N | N |
+| 并行 DDL |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [添加索引加速](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [元数据锁](/metadata-lock.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [`FLASHBACK CLUSTER`](/sql-statements/sql-statement-flashback-cluster.md) |  Y  |  Y  | Y | Y | N | N | N | N | N | N | N |
+| [暂停](/sql-statements/sql-statement-admin-pause-ddl.md)/[恢复](/sql-statements/sql-statement-admin-resume-ddl.md) DDL | Y | Y | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:133]:
```suggestion
| [TiDB DDL V2 架构](/ddl-v2.md) | E | N | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md (hunk 3 above)
Comment (qiancai) [reply basic-features.md:218]: /dm/dm-overview.md

File: basic-features.md
Comment (qiancai) [inline basic-features.md:1]: preview: https://pingcap-docsite-preview-git-v760-release-notes-qiancai.vercel.app/zh/tidb/dev/basic-features

**Review by hfxsd — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: basic-features.md [hunk 5]
```diff
@@ -20,235 +20,240 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) [^5] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^6] |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) [^7] | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiCDC 支持保存数据到存储服务 (Amazon S3/GCS/Azure Blob Storage/NFS)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | Y | Y | E | N | N | N | N | N | N | N |
+| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [DM](/dm/dm-overview.md) 支持迁移 MySQL 8.0  | Y | E | E | E | E | N | N | N | N | N | N |
 ## 管理，可视化和工具
```
Comment (Oreoxmt) [inline basic-features.md:220]:
```suggestion

## 管理，可视化和工具
```

File: basic-features.md [hunk 6]
```diff
@@ -20,235 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [索引合并](/explain-index-merge.md) |  Y  | Y | Y | Y | Y | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) |  Y  | Y | Y | Y | E | E | N | N | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) |  E  | E | E | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) |  Y  | N | N | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 7.6 | 7.5 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) |  Y  |  Y  | Y | Y | Y | Y | Y | E | E | E | E |
+| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) |  Y  |  Y  | E | N | N | N | N | N | N | N | N |
+| [执行计划绑定 (SQL Binding)](/sql-plan-management.md#执行计划绑定-sql-binding) |  Y  |  Y  | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [跨数据库执行计划绑定 (Cross-DB Binding)](/sql-plan-management.md#跨数据库执行计划绑定-cross-db-binding) |  Y  |  N  | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:97]:
```suggestion
| [跨数据库执行计划绑定 (Cross-DB Binding)](/sql-plan-management.md#跨数据库绑定执行计划-cross-db-binding) |  Y  |  N  | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 7]
```diff
@@ -20,235 +20,241 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | N | N | N | N | N | N | N | N | N |
+| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | E | E | N | N | N | N | N | N | N | N | N |
+| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | Y | Y | E | N | N | N | N | N | N | N | N |
+| [选择执行分布式执行框架任务的 TiDB 节点](/system-variables.md#tidb_service_scope-从-v740-版本开始引入) | Y | Y | N | N | N | N | N | N | N | N | N |
+| 通过系统变量 [`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-从-v530-版本开始引入) 控制 PD Follower Proxy 功能 | Y | Y | Y | Y | Y | Y | Y | N | N | N | N |
+| 通过系统变量 [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-从-v760-版本开始引入) 控制 [Active PD Follower](/tune-region-performance.md#通过-active-pd-follower-提升-pd-region-路由信息查询的服务能力) 功能 | E | N | N | N | N | N | N | N | N | N | N |
```
Comment (qiancai) [inline basic-features.md:257]:
```suggestion
| 通过系统变量 [`pd_enable_follower_handle_region`](/system-variables.md#pd_enable_follower_handle_region-从-v760-版本开始引入) 控制 [Active PD Follower](/tune-region-performance.md#通过-active-pd-follower-提升-pd-region-信息查询服务的扩展能力) 功能 | E | N | N | N | N | N | N | N | N | N | N |
```

**Pull request merged**